bundles/helm/Dockerfile

```diff
@@ -1,6 +1,5 @@
 FROM ubuntu:18.04
-RUN apt-get update
-RUN apt-get -y install curl
+RUN apt-get update && apt-get -y install curl
```

Comment:
Not an optimization, it wasn't building for me until I combined into a single line